docs/docusaurus/docs/guides/expectations/components_how_to_create_and_edit_expectations_with_instant_feedback_from_a_sample_batch_of_data/_preface.mdx

```diff
@@ -10,3 +10,39 @@ This guide will take you through the process of creating <TechnicalTag tag="expe
 - [Created a Datasource](/docs/guides/connecting_to_your_data/connect_to_data_overview).
 
 </Prerequisites>
+
+## Steps
```

Comment:
```suggestion
## Steps
```
I really dislike _Steps_.  It's a personal bias. What will users achieve after completing this procedure? Create a meaningful heading.

Reply:
Let's set up an information architecture review meeting and revise our standards for formatting how-to guides -- making a change to the current standards is out of scope for this PR.